tinywallet/pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[tool.poetry]
+name = "tinywallet"
+version = "0.1.0"
+description = "A Python 3 Decred wallet."
+license = "ISC"
+homepage = "https://decred.org/"
+repository = "https://github.com/decred/tinydecred/"
+documentation = "https://github.com/decred/tinydecred/blob/master/tinywallet"
+authors = [
+    "Brian Stafford <https://github.com/buck54321>",
+    "The Decred developers <https://decred.org/community>"
+]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Environment :: X11 Applications :: Qt",
+    "Intended Audience :: End Users/Desktop",
+    "License :: OSI Approved :: ISC License (ISCL)",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.7",
+    "Topic :: Office/Business :: Financial"
+]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+decred = {path = "../decred/"}
```

Comment:
This dependency will continue as relative import or there will be a time when it's from the pip repository?

Reply:
That will become a version number once we have the package up on the Package Index. We're asked for the `decred` package name there, see pypa/pypi-support#163 .